moulinette/core.py

```diff
@@ -573,7 +573,7 @@ def acquire(self):
                     self._lock()
                     break
 
-            if (time.time() - start_time) > self.timeout:
+            if self.timeout is not None and (time.time() - start_time) > self.timeout:
```

Comment:
I was just wondering if it makes sense that timeout 0 is **also** a case for "no timeout"...